src/machines/java/com/enderio/machines/client/gui/screen/DrainScreen.java

```diff
@@ -39,9 +41,10 @@ protected void init() {
             () -> menu.getBlockEntity().isRangeVisible(), state -> menu.getBlockEntity().setIsRangeVisible(state),
             () -> menu.getBlockEntity().isRangeVisible() ? EIOLang.HIDE_RANGE : EIOLang.SHOW_RANGE));
 
-            // TODO: NEO-PORT: ImageButton's
-//        addRenderableWidget(new ImageButton(leftPos + imageWidth - 2*16, topPos + 2 + 16*2, 8, 8, 8, 0, 16, BUTTONS, (b) -> menu.getBlockEntity().increaseRange()));
-//        addRenderableWidget(new ImageButton(leftPos + imageWidth - 2*16, topPos + 2 + 16*2 + 8, 8, 8, 8, 8, 16, BUTTONS, (b) -> menu.getBlockEntity().decreaseRange()));
+        addRenderableWidget(new EIOImageButton(this, leftPos + imageWidth - 2 * 16, topPos + 2 + 16 * 2, 8, 8, new WidgetSprites(PLUS, PLUS),
```

Comment:
Could we put all the new WidgetSprites things into a static final everywhere? Feels a bit nicer than constructing inline, thanks!